runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperPlugin.java

```diff
@@ -238,6 +238,8 @@ void setupSwaps()
 		swap("big net", "harpoon", config::swapHarpoon);
 		swap("net", "harpoon", config::swapHarpoon);
 
+		swap("lure", "bait", config::swapBait);
```

Comment:
Cave eels also have "Bait" options, as do some mystery Net / Bait spots. They should be covered the same way as the harpoon spots.